cdc/processor/manager_test.go

```diff
@@ -305,7 +305,7 @@ func TestQueryTableCount(t *testing.T) {
 	m := NewManager(&model.CaptureInfo{ID: "capture-test"}, nil, &liveness, cfg).(*managerImpl)
 	ctx := context.TODO()
 	// Add some tables to processor.
-	tables := spanz.NewMap[tablepb.TablePipeline]()
+	tables := spanz.NewHashMap[tablepb.TablePipeline]()
```

Comment:
What is the meaning of the package name `spanz`?

Reply:
It should be `spans` (just like the `strings` package), but it may cause naming confliction, there are so many spans variables. So the name `spanz`, easy to type and easy to pronounce.